srtcore/core.cpp

```diff
@@ -5529,6 +5544,8 @@ void * srt::CUDT::tsbpd(void* param)
             tsNextDelivery = steady_clock::time_point(); // Ready to read, nothing to wait for.
         }
 
+        SRT_ATR_UNUSED bool wakeup_on_signal = true;
```

Comment:
```suggestion
        SRT_ATR_UNUSED bool bWakeupOnSignal = true;
```

Reply:
Not sure if that convention is really required on local variables, but ok. Still, it's in use for logs only.